common/src/main/resources/less/style.less

```diff
@@ -207,3 +207,21 @@ body {
 .dragging-over {
   background-color: gray;
 }
+
+.elevation-plot {
+  height: 300px;
+}
+
+.site-toggle {
+  margin-left: 10px;
+  margin-right: 10px;
+  padding-top: 6px;
+}
+
+.site-toggle.ui.slider.checkbox .box:before, .site-toggle.ui.slider.checkbox label:before {
+  background-color: #545454 !important;
```

Comment:
Even being that specific you need an `!important`?

Reply:
The setting we are overriding has `!important` in SUI, so I couldn't find a way to override it otherwise.

Reply:
ah ok, that makes sense